packages/adapters/wagmi/src/connectors/AuthConnector.ts

```diff
@@ -68,12 +73,14 @@ export function authConnector(parameters: AuthParameters) {
       await provider.disconnect()
     },
 
-    async getAccounts() {
-      const provider = await this.getProvider()
-      const { address } = await provider.connect()
-      config.emitter.emit('change', { accounts: [address as Address] })
```

Comment:
We've been calling `provider.connect` again which does more network requests, instead we can store the accounts internally.